L/libisal/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libisal"
+version = v"2.30.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/intel/isa-l.git", "2bbce31943289d5696bcf2a433124c50928226a2")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/isa-l/
+./autogen.sh 
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "musl")
+]
+
```

Comment:
Maybe this can work on other platforms?
```suggestion
platforms = supported_platforms()
```

Reply:
I tried to build for all platforms. On aarch it complains about unknown mnemonics and on Apple and Windows it complains either not to have found NASM or not in a recent version (>= 2.11.01). I checked, and NASM_jll should be available for all platforms in version 2.15. I dont know how to address this.

Reply:
I also tried with both, NASM_jll and YASM_jll and for those platforms, where NASM isnt detected properly, detection of YASM also fails.

Reply:
For Windows, there is an alternative Makefile and it should be used like this:
`nmake -f Makefile.nmake`
But I also did not succeed to make this work.

Reply:
For aarch64 you need gcc 8, because it's trying to compile for SVE.

Reply:
On Windows it finds yasm no problem for me, but it fails to compile some avx512 assembly